apps/dashboard/app/helpers/batch_connect/session_contexts_helper.rb

```diff
@@ -1,23 +1,31 @@
 module BatchConnect::SessionContextsHelper
   def create_widget(form, attrib, format: nil)
-    return "" if attrib.fixed?
+    return '' if attrib.fixed?
 
     widget = attrib.widget
     field_options = attrib.field_options(fmt: format)
-    all_options = attrib.all_options(fmt: format)
-
+    all_options = attrib.all_options(fmt: format) 
+  
```

Comment:
Be careful of extra whitespace here. In a `git diff` it'll be highlighted as red.